pkgs/development/python-modules/statmake/default.nix

```diff
@@ -24,6 +24,13 @@ buildPythonPackage rec {
     sha256 = "OXhoQAD4LEh80iRUZE2z8sCtWJDv/bSo0bwHbOOPVE0=";
   };
 
+  postPatch = ''
```

Comment:
Why move `postPatch` up? It is lower level action executed with other phases so so far we have been always moving it down (see also https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887).

Reply:
Seeing that post for the first time. So far everything I have seen is ordered in the order how it is run, not some higher, middle lower level ordering which is very abstract. I can drop this again and we move this discussion to another time.